kobuki/03660af/03660af.bug

```diff
@@ -85,4 +85,4 @@ fault-codes:
   - CONCURRENCY:NO-SYNC
 failure-codes:
   - SOFTWARE:LIVENESS
-  - SYSTEM:LOSS-OF-CONTROL
+  - SYSTEM:UNRESPONSIVE
```

Comment:
Is there a case for BEHAVING INCORRECTLY here as well?

Reply:
From today's discussion: Let's keep BEHAVING-INCORRECTLY as the only label. UNRESPONSIVE is a bit of a stretch.